scripts/qa_crowbarsetup.sh

```diff
@@ -2436,7 +2436,7 @@ function enable_ssl_generic()
             return
         ;;
         heat)
-            if [[ ! $(iscloudver 7plus) ]] ; then
+            if ! iscloudver 7plus ; then
```

Comment:
would be better to have a normal check, rather than a negation check with a return so its simpler to read.

    if iscloudver 7plus ; then
      $p "$a['api']['protocol']" "'https'"
    fi

as I assume that the default for ['api']['protocol'] is "http" and only in the case of the cloud version being 7plus we enable https?